controls/private_repo.sp

```diff
@@ -0,0 +1,215 @@
+locals {
+  include_forks = "false"
+  private_repo_best_practices_common_tags = merge(local.github_insights_common_tags, {
+    service = "GitHub/Repository"
+  })
+}
+
+benchmark "private_repo_best_practices" {
```

Comment:
Do we still need 2 separate benchmarks for private vs. public repos? How many checks are exclusive to private or public vs. being in both benchmarks?

Reply:
@cbruno10  Currently we have 7 controls for private repo and 14 controls for public repo. I believe it's logical to continue with the current strategy and maintain distinct benchmarks for each type of repository.